### PR TITLE
implemented CSkidmarks

### PR DESCRIPTION
**Pull request by aap**

### CODE REVIEW

File: src/render/Skidmarks.cpp [hunk 1]
```diff
@@ -1,12 +1,247 @@
...
 #include "Skidmarks.h"
 
-WRAPPER void CSkidmarks::Clear(void) { EAXJMP(0x518130); }
-WRAPPER void CSkidmarks::Update() { EAXJMP(0x518200); }
+CSkidmark CSkidmarks::aSkidmarks[NUMSKIDMARKS];
 
-WRAPPER void CSkidmarks::Render(void) { EAXJMP(0x5182E0); }
-WRAPPER void CSkidmarks::RegisterOne(uint32 id, CVector pos, float fwdx, float fwdY, bool *isMuddy, bool *isBloddy) { EAXJMP(0x5185C0); }
+RwImVertexIndex SkidmarkIndexList[16 * 6];
+RwIm3DVertex SkidmarkVertices[16 * 2];
```
Comment (ShFil119) [inline src/render/Skidmarks.cpp:12]: aren't they two dimensions arrays?

Comment (aap) [reply src/render/Skidmarks.cpp:12]: the RW functions take pointers...so i'd say no. but semantically yes.

Comment (ShFil119) [reply src/render/Skidmarks.cpp:12]: btw names for sizes (16, 6 and 2) would be nice, these values are used for loops.

Comment (aap) [reply src/render/Skidmarks.cpp]: eh, ok. we have SKIDMARK_LENGTH now.

**Pull request merged**